bin/create-cluster.sh

```diff
@@ -14,8 +14,9 @@ echo "Primary Zone = $GKE_PRIMARY_ZONE"
 echo "Additional Zones = $GKE_NODE_LOCATIONS"
 echo "Cluster Name = $GKE_CLUSTER_NAME"
 echo "Cluster Namespace = $GKE_CLUSTER_NS"
+echo "Cluster Monitoring Namespace = $GKE_MONITORING_NS"
```

Comment:
Lee, I think you want to add an entry in ../etc/gke-env.template.cfg for $GKE_MONITORING_NS as well.

Reply:
the  ../etc/gke-env.template.cfg file is actually a local file and is ignored by helm. That way you don't end up committing your own changes to the repo.  Just copy the template file again

Reply:
Wajih's directions for creating a cluster are to take gke-env.template.cfg and copy it to gke-env.cfg, and make modifications for your environment. So the template file should contain all the variables you need to set when deploying the cluster.

Reply:
sorry, I was confused because you added cfg at the end.   So yes it should be in the gke-env.template file.  But on the other hand, it works without it as it appears Helm now creates the namespace if it doesn't exist.  I'm sure if never did this.  I'll add it in as a safety net anyway.  It was there originally, not sure what happened.